### PR TITLE
issue#331_PlantUML for hand draw diagram

### PR DESCRIPTION
**Pull request by xiangli6**
Issue#331 
section 2
Create a PlantUML class diagram that is like the hand-draw block diagram

### CODE REVIEW

File: docs/Developer/ClassDiagrams/packages.puml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+skinparam linetype ortho
+/' hide the class members '/
+hide members
+
+/' packages '/
+package Core{
+	/' class '/
+   class Simulator
+   class CPUSpikingModel implements Model
+	class GPUSpilingModel implements Model
```
Comment (stiber) [inline docs/Developer/ClassDiagrams/packages.puml:13]: Typo "GPUSpikingModel"

File: docs/Developer/ClassDiagrams/packages.puml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	class Operations
+	class OperationManager
+	class ParameterManager
+
+	/' class relationships '/
+	Model -left-o Simulator 
+	Operations o-right- OperationManager
```
Comment (stiber) [inline docs/Developer/ClassDiagrams/packages.puml:20]: `OperationsManager` contains `Operations`, right?

File: docs/Developer/ClassDiagrams/packages.puml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+	class OperationManager
+	class ParameterManager
+
+	/' class relationships '/
+	Model -left-o Simulator 
+	Operations o-right- OperationManager   
+	Simulator ---> OperationManager 
+
+	/' relative position of classes in this package'/
+	CPUSpikingModel-[hidden]right>GPUSpilingModel
```
Comment (stiber) [inline docs/Developer/ClassDiagrams/packages.puml:24]: Typo "`GPUSpikingModel`"

**Review by stiber — CHANGES_REQUESTED** [review]
This looks great! Just a couple typos.

File: docs/Developer/ClassDiagrams/packages.puml (hunk 2 above)
Comment (xiangli6) [reply docs/Developer/ClassDiagrams/packages.puml:20]: From the hand_draw diagram, it looks like the Operations contains OperationsManager? The diamond of UML arrow in this diagram is next to Operations.

Comment (stiber) [reply docs/Developer/ClassDiagrams/packages.puml:20]: You found an error in the hand drawn diagram that nobody has noticed! That's great; and we can fix it here.

**Review by stiber — APPROVED** [review]
Looks good!

**Pull request merged**